src/PeopleSDKAdapter.js

```diff
@@ -184,4 +185,23 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
 
     return this.getPersonObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits people list
+   *
+   * @param {string} query search query
+   * @returns {external:Observable.<Person>} Observable that emits person list based on search query
+   */
+  getPeopleList(query) {
```

Comment:
don't forget to use the logger:
- at the start of the function (getPeopleList() called)
- when emitting something (observer.next)
- when receiving an error (catch)